### ISSUE TITLE
known_atoms option

### ISSUE THREAD

**Issue by frepond** [msgpack/msgpack-erlang#68]
Hi, according to documentation "msgpacked binaries are decoded as atoms with `erlang:binary_to_existing_atom/2` with encoding utf8". It's not working for me and looking at the code I couldn't find any call to `erlang:binary_to_existing_atom/2`. Am I missing something?

Kind regards

**Comment by frepond** [comment]
I would suggest using a map for `know_atoms` and then modifying `msgpack_unpacker:maybe_bin` as follow:

```erlang
maybe_bin(Bin, ?OPTION{known_atoms=KnownAtoms}) ->
    maps:get(Bin, KnownAtoms, Bin).
```

**Comment by kuenishi** [comment]
I thought I've already implemented that, but somehow it's gone (from my head). You're missing nothing; it's missing. I discussed it with @mururu yesterday and will find a resolution for this soon. Thank you for pointing this out.

**Comment by cleverfox** [comment]
Any news about this issue? Still no code for decoding atoms.

**Comment by kuenishi** [comment]
Thanks to @cleverfox at https://github.com/msgpack/msgpack-erlang/pull/77, this issue was fixed as of 0.8.0.
